presto-main/src/main/java/com/facebook/presto/sql/planner/PlannerUtils.java

```diff
@@ -400,4 +410,84 @@ public static RowExpression coalesce(List<RowExpression> expressions)
     {
         return new SpecialFormExpression(SpecialFormExpression.Form.COALESCE, expressions.get(0).getType(), expressions);
     }
+
+    public static AggregationNode.Aggregation removeFilterAndMask(AggregationNode.Aggregation aggregation)
```

Comment:
I just realized there is a separate AggregationNodeUtils class. WE can move these functions into that class.